src/app/components/balance/balance-private.tsx

```diff
@@ -0,0 +1,25 @@
+import { type HTMLStyledProps } from 'leather-styles/jsx';
+
+import { useTogglePrivateMode } from '@app/store/settings/settings.actions';
+import { useIsPrivateMode } from '@app/store/settings/settings.selectors';
+import { PrivateTextLayout } from '@app/ui/components/privacy/private-text.layout';
+
+interface BalancePrivateProps extends HTMLStyledProps<'span'> {
+  children: React.ReactNode;
+  canClickToShow?: boolean;
+}
+
+export function BalancePrivate({ children, canClickToShow, ...rest }: BalancePrivateProps) {
```

Comment:
Maybe this can be `PrivateBalance` to match the convention of `PrivateText` ? 

Do you need the prop `canClickToShow`? It seems to be always passed in as `true`?

Reply:
I renamed the component and am now using it in a place that needs to set the prop `canClickToShow=false` (I actually don't pass it at all).